package.json

```diff
@@ -82,6 +82,7 @@
   "license": "AGPL",
   "dependencies": {
     "@hpcc-js/wasm": "^2.16.0",
+    "@maplibre/maplibre-gl-leaflet": "0.0.21",
```

Comment:
This compatibility shim works, but it preserves the Leaflet user experience, which is much less fluid than MapLibre GL JS. Migrating from Leaflet to MapLibre GL JS would enable smooth zooming, rotation, and better rendering performance. Something to consider for the future.